Pinta.Effects/Effects/LocalHistogram.cs

```diff
@@ -33,12 +33,44 @@ private static void SetToZero (Span<int> data)
 		data.Clear ();
 	}
 
-	public virtual ColorBgra Apply (in ColorBgra src, int area, Span<int> hb, Span<int> hg, Span<int> hr, Span<int> ha)
-		=> src;
+	public delegate ColorBgra Applier (
+		ColorBgra src,
+		int area,
+		Span<int> hb,
+		Span<int> hg,
+		Span<int> hr,
+		Span<int> ha);
+
+	public delegate ColorBgra AlphaApplier (
+		ColorBgra src,
+		int area,
+		int sum,
+		Span<int> hb,
+		Span<int> hg,
+		Span<int> hr);
 
-	//same as Apply, except the histogram is alpha-weighted instead of keeping a separate alpha channel histogram.
-	public virtual ColorBgra ApplyWithAlpha (in ColorBgra src, int area, int sum, Span<int> hb, Span<int> hg, Span<int> hr)
-		=> src;
+	/// <summary>
+	/// Default implementation of <see cref="Applier"/>
+	/// </summary>
+	public static ColorBgra ReturnOriginal (
```

Comment:
Are these unused?

Reply:
@cameronwhite yes, they are unused. They are there to provide placeholder methods for those who need it. I thought they would be needed for refactoring but that wasn't the case.

Reply:
Yeah, I think we should just remove them if none of the existing implementations needed them.

Reply:
@cameronwhite I just removed them